vta/tests/python/de10nano/test_program_rpc.py

```diff
@@ -0,0 +1,45 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import sys, os
+import tvm
+from tvm import rpc
+from vta import get_bitstream_path, download_bitstream, program_fpga, reconfig_runtime
+
+host = os.environ.get("VTA_PYNQ_RPC_HOST", "de10nano")
```

Comment:
@tmoreau89 Shall we rename this environment variable for targets other than PYNQ?

Reply:
I was also confused when I first browsed the code.
I would have expected something like:
VTA_RPC_HOST = {pynq|de10nano|ultra96|etc}
VTA_RPC_PORT = 9091

Reply:
I think I've been using this variable to be able to target and program different FPGAs. For in that spririt, I would use `VTA_DE10_RPC_HOST` for this test. Your bashrc could contain multiple hosts including `VTA_PYNQ_RPC_HOST`, `VTA_DE10_RPC_HOST`, `VTA_ULTRA96_RPC_HOST` etc.

Reply:
Makes perfect sense, perhaps coupled with your idea of folding fpga target specific info in environment.py.
Right now introducing VTA_DE10_RPC_HOST anywhere VTA_PYNQ_RPC_HOST, including matrix_multiply.py for instance would add a lot of boiler plate code.

Reply:
I see; upon second though it would be cleaner to use `VTA_RPC_HOST` environment variables given that some of our unit tests assume that we're targeting the pynq boards.

Reply:
I think that's totally acceptable as long as we update documentation too to reflect that change in requirements. Grep-ing for `VTA_PYNQ_RPC_HOST` should do the trick

Reply:
Agree!